app/helpers/import_topics_helper.rb

```diff
@@ -7,6 +7,13 @@ def self.define_attributes(columns)
     attributes["topic_name"] = columns[1].strip
     attributes["max_choosers"] = columns[2].strip
     attributes["category"] = columns[3].strip if columns.length > 3
+    define_attributes_extra(attributes, columns)
+    attributes
+  end
+
+  def self.define_attributes_extra(attributes, columns)
```

Comment:
Code Climate was giving me an issue about more number of Assignment Branch Condition size here. You can see that I had not done this earlier. I only did this after Code Climate complained about it.